src/player.cpp

```diff
@@ -106,6 +106,9 @@ namespace Player {
 	int message_box_offset_x = (screen_width - MENU_WIDTH) / 2;
 	bool has_custom_resolution = false;
 
+	int32_t iniVars_min = 0;
```

Comment:
not sure if ini_vars_min and ini_vars_max is better for consistency (instead of mixed underscores and camel case)

Reply:
sure, I paused this one after wondering if #3145 would be a better solution: